javascript/domUpdates.js

```diff
@@ -0,0 +1,21 @@
+const domUpdates = {
+  displayPlayers(player1, player2, player3) {
+    let playerOneCard = document.querySelector('.player-one-card');
+    let playerTwoCard = document.querySelector('.player-two-card');
+    let playerThreeCard = document.querySelector('.player-three-card');
+    playerOneCard.innerText = player1.name
+    playerTwoCard.innerText = player2.name
+    playerThreeCard.innerText = player3.name
+  }
```

Comment:
This method looks like a good opportunity for DRYing up your code -- what if all the player elements had the same class on them - `player`, and you selected all of them at once with `querySelectorAll`? Then you could loop through that result and set the `innerText` of each one with a single line of code.